src/replica/split/replica_split_manager.h

```diff
@@ -33,6 +33,12 @@ class replica_split_manager : replica_base
     int32_t get_partition_version() const { return _partition_version.load(); }
     gpid get_child_gpid() const { return _child_gpid; }
     void set_child_gpid(gpid pid) { _child_gpid = pid; }
+    bool is_splitting() const
+    {
+        return _child_gpid.get_app_id() > 0 && _child_init_ballot > 0 &&
+               _split_status == split_status::SPLITTING;
+    }
+    split_status::type get_meta_split_status() { return _meta_split_status; }
```

Comment:
Why don't simply call it `get_split_status`?

Reply:
There are two different status, `_split_status` is replica local split status, and `_meta_split_status` is split_status from meta server through `on_config_sync`.